internal/twitchapiclient/client.go

```diff
@@ -0,0 +1,36 @@
+package twitchapiclient
+
+import (
+	"errors"
+
+	"github.com/Chatterino/api/pkg/config"
+	"github.com/nicklaw5/helix"
+)
+
+// New returns a helix.Client that has requested an AppAccessToken and will keep it refreshed every 24h
+func New(cfg config.APIConfig) (*helix.Client, error) {
+	if cfg.TwitchClientID == "" {
+		return nil, errors.New("twitch-client-id is missing, can't make Twitch requests")
+	}
+
+	if cfg.TwitchClientSecret == "" {
+		return nil, errors.New("twitch-client-secret is missing, can't make Twitch requests")
```

Comment:
small nitpick, but you could prefix these errors with `[Config]` since this is related to missing config entries.

Reply:
:nerd_face: Go errors shouldn't have any sort of categorization, rather it's up to the error consumer to spit out an error if needed